desec.py

```diff
@@ -1180,6 +1204,14 @@ def main():
             with open(arguments.file, 'w') as f:
                 json.dump(data, f)
 
+        elif arguments.action == 'export-zone':
+
+            print("Export Zone File")
```

Comment:
Please remove this statement. The JSON-based `export` does not print anything either.

Reply:
Done.